### ISSUE TITLE
train_cls

### ISSUE THREAD

**Issue by yolanda3212** [Juliachang/SC-CAM#15]
Is the author's classification based on siamese network?

**Comment by wasidennis** [comment]
no, it's the typical classification network but with two heads: parent classifier and sub-category classifier.
